mautic/mautic
# Report CSV export with enclosures

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Some clients required exports with empty columns with enclosures. 
This PR add support for it as option in Configuration > Report Settings tab.

![image](https://user-images.githubusercontent.com/462477/50448631-97b0df80-0922-11e9-97e4-18b27ef0b5d3.png)


Old format:

`Firstname, "some text",,Lastname`

New format:

`"Firstname","some test","","Lastname"`


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.  Try CSV export contacts with some contacts etc. 
3. Should works like before
4. Enable Always enclose data in CSV export in Configurations > Report Settings tab
5. Export CSV report
6. See all fields are enclosures, even empty fields
